platform/linux-generic/_ishm.c

```diff
@@ -864,8 +864,8 @@ int _odp_ishm_reserve(const char *name, uint64_t size, int fd,
 		 * the same address every where, otherwise alignment may be
 		 * be wrong for some process */
 		hp_align = align;
-		if (hp_align <= odp_sys_huge_page_size())
-			hp_align = odp_sys_huge_page_size();
+		if (hp_align <= page_hp_size)
+			hp_align = page_hp_size;
```

Comment:
Presumably this has nothing to do with the bug fix and is just an efficiency improvement since on Line 804 above we have `page_hp_size = odp_sys_huge_page_size();`.

Reply:
Correct